src/manipulation/wrapMap.js

```diff
@@ -3,12 +3,11 @@ define( function() {
 // We have to close these tags to support XHTML (#13200)
 var wrapMap = {
 
-	// Support: IE9
```

Comment:
We don't run tests in XHTML so them not failing doesn't prove anything. As the comment below states, XHTML parsers require certain structure, e.g. omitting `<tbody>` is not allowed. I'm just asking - isn't that the same case for `<option>`? Isn't that required to be contained inside `<select>` in XHTML?